.tool-versions

```diff
@@ -1 +1 @@
-nodejs 16.17.0
+nodejs 22.0.0
```

Comment:
20 as we are using node20 in the action?

Reply:
Fixed and test with test branch.